src/khoj/database/models/__init__.py

```diff
@@ -263,6 +265,11 @@ class UserSearchModelConfig(BaseModel):
     setting = models.ForeignKey(SearchModelConfig, on_delete=models.CASCADE)
 
 
+class UserPaintModelConfig(BaseModel):
```

Comment:
Let's call this `UserImageModelConfig` for clarity? It'll make the relationship between this object and `TextToImageModelConfig` much more obvious.

```suggestion
class UserImageModelConfig(BaseModel):
```

Reply:
Renamed as requested